awesome.c

```diff
@@ -700,12 +701,12 @@ main(int argc, char **argv)
 
         if(!success)
         {
-            fprintf(stderr, "✘ Configuration file syntax error.\n");
+            fprintf(stdout, "ERROR\n");
```

Comment:
I think "ERROR: " could be used as a prefix with the error above already.
Then it would only return here.

Reply:
could you insert is as a "Suggestion" here?
i'm not sure i'm following

Reply:
Remove the line above, and add "ERROR: " in front of `fprintf(stderr, "%s\n", err);` (I cannot make a suggestion there).
=> `fprintf(stderr, "ERROR: %s\n", err);`

Reply:
do you mean like this?